server/routes/api/__test__/user.unit.test.js

```diff
@@ -1,7 +1,86 @@
 const request = require('supertest');
 const app = require('../../../../app');
+const User = require('../../../models/user');
+
 describe('User Routes', () => {
-  test('GET method', done => {
-    done();
+  let userID;
+  afterAll(() => {
+    if (userID) {
+      // Delete test user after create test
+      User.findByIdAndDelete({ _id: userID }).exec();
+    }
+  });
+
+  describe('POST /user', () => {
+    it('will return error without payload', async () => {
+      const res = await request(app)
+        .post('/api/user')
+        .type('form'); // send post with no payload
+      const errorObj = res.body.error.errors;
+      expect(res.statusCode).toEqual(400);
+      expect(Object.keys(errorObj)).toEqual(['email', 'password']); // should return two errors
+      expect(errorObj.email.message).toBe('You must provide an email address.');
+      expect(errorObj.password.message).toBe('You must provide a password.');
+    });
+
+    it('will return error without an email', async () => {
+      const res = await request(app)
+        .post('/api/user')
+        .type('form')
+        .send('password=test'); // send post with no email
```

Comment:
This format _works_. I'm curious where exactly this gets converted, but this can get kind of messy when we need to send multiple pieces of data. Let's switch these strings to something like:

```
.send({
  password: test
});
```

Reply:
Updated for all tests sending data.